### PR TITLE
[1.14] exec: Close pipe fds to prevent hangs

### PR DESCRIPTION
**Pull request by haircommander**
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

File: oci/runtime_oci.go [hunk 1]
```diff
@@ -305,6 +309,13 @@ func (r *runtimeOCI) ExecContainer(c *Container, cmd []string, stdin io.Reader,
 		}
 
 		cmdErr = execCmd.Run()
+
+		// The read side of the pipe should be closed after the container process has been started.
```
Comment (mrunalp) [inline oci/runtime_oci.go]: This looks incorrect. Did we miss the start/wait split here?

Comment (haircommander) [reply oci/runtime_oci.go]: ah, oops, we sure did

Comment (haircommander) [reply oci/runtime_oci.go]: fixed!

**Pull request merged**